etc/docs/template/layouts/partials/assets/javascripts.html

```diff
@@ -0,0 +1,7 @@
+<script type="text/javascript" src="{{.Site.BaseURL}}/s/js/jquery.js"></script>
+<script type="text/javascript" src="{{.Site.BaseURL}}/s/lib/bootstrap.js"></script>
+<script type="text/javascript" src="{{.Site.BaseURL}}/s/lib/highlight/highlight.pack.js"></script>
```

Comment:
Do these get created by the generation of the site? I'm not seeing any of the `/lib/` scripts in this PR

Reply:
No.  These scripts are present in the gh-pages branch (https://github.com/mongodb/node-mongodb-native/tree/gh-pages/s/lib).  They'll be present after part 2 of this PR, when I move the contents of `gh-pages` to main.